backend/app/infrastructure/repositories/context.py

```diff
@@ -27,3 +28,44 @@ def get_real_round_id(self, context_id: int) -> int:
             .first()
         )
         return instance.r_realid
+
+    def get_random(self, real_round_id: int, n: int = 1, tags=None):
```

Comment:
If the limit n (number of contexts) is 1 for all current challenges, I think it is a better idea to remove this parameter.

Reply:
Great idea. I just kept the first output instead.